Xbim.Ifc/IfcStore.cs

```diff
@@ -66,10 +68,18 @@ public class IfcStore : IModel, IDisposable, IFederatedModel, IEquatable<IModel>
         
         private readonly ReferencedModelCollection _referencedModels = new ReferencedModelCollection();
 
+        static IfcStore()
+        {
+
+            XbimServices.Current.ConfigureServices(s => s.AddXbimToolkit());

```

Comment:
This allows consumers to use IfcStore out the box without configuring DI.
We could possibly get smarter and determine if DI had been configured